gtk/src/light/gtk-3.20/_drawing.scss

```diff
@@ -709,8 +709,8 @@
     margin: 2px;
     min-height: 16px;
     min-width: 16px;
-    outline-color: $focus_color;
-    outline-offset: -2px;
+    outline-color: transparentize(black, 0.6);
+    outline-offset: -3px;
```

Comment:
Why changing the switch outline offset? Testing it, it looks too far from the border to me